_search-plugins/cross-cluster-search.md

```diff
@@ -243,3 +243,56 @@ curl -XGET -k -u booksuser:password 'https://localhost:9250/opensearch-ccs-clust
   }]
 }
 ```
+
+## Sample bare metal/virtual machines setup
+
+If you are running OpenSearch on a bare metal server or using a virtual machine, you can run the same commands, specifying the IP (or domain) of the OpenSearch cluster.
+For example, in order to configure remote cluster for Cross-cluster search, find the IP of the remote node or domain of the remote cluster and run command:
+
+```json
+curl -k -XPUT -H 'Content-Type: application/json' -u 'admin:<custom-admin-password>' 'https://opensearch-domain-1:9200/_cluster/settings' -d '
+{
+  "persistent": {
+    "cluster.remote": {
+      "opensearch-ccs-cluster2": {
+        "seeds": ["opensearch-domain-2:9300"]
+      }
+    }
+  }
+}'
+```
+It is enough to point to only one of the node IPs on the remote cluster, as node discovery process will query all nodes in that cluster.
+{: .tip }
+
+You can now run queries across both clusters.
+
+```bash
+curl -XGET -k -u 'admin:<custom-admin-password>' 'https://opensearch-domain-1:9200/opensearch-ccs-cluster2:books/_search?pretty'
+{
+  ...
+  "hits": [{
+    "_index": "opensearch-ccs-cluster2:books",
+    "_id": "1",
+    "_score": 1.0,
+    "_source": {
+      "Dracula": "Bram Stoker"
+    }
+  }]
+}
+```
+
+## Sample Kubernetes/Helm setup
+If you are using Kubernetes clusters to deploy OpenSearch, you have to configure the remote cluster using the `LoadBalancer` or `Ingress` services. The services that get created using the following [Helm]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/helm/) example are of the `ClusterIP` type and are only be accessible inside the cluster, therefore external services need to be used.
```

Comment:
Second sentence: Given that we already refer to services in both the first and third sentences, should we use a different word in "The services created"?

Reply:
I've reworded this slightly to:
```
## Sample Kubernetes/Helm setup
If you are using Kubernetes clusters to deploy OpenSearch, you need to configure the remote cluster using either the `LoadBalancer` or `Ingress`. The kubernetes services created using the following [Helm]({{site.url}}{{site.baseurl}}/install-and-configure/install-opensearch/helm/) example are of the `ClusterIP` type and are only accessible from within the cluster; therefore, you must use an externally accessible endpoint:
```

Reply:
@AntonEliatra Looks good, but the second instance of "Kubernetes" should be capitalized.

Reply:
@natebower thats fixed now